eppo/src/main/java/cloud/eppo/android/EppoClient.java

```diff
@@ -354,6 +354,21 @@ public JSONObject getJSONAssignment(String flagKey, String subjectKey, JSONObjec
     return getJSONAssignment(flagKey, subjectKey, new SubjectAttributes(), defaultValue);
   }
 
+  public String getJSONStringAssignment(String flagKey, String subjectKey, String defaultValue) {
```

Comment:
We have two signatures for each assignment getter--one that takes subject attributes and one that doesn't. You dropped this one right in the middle of the two `getJSONAssignment` definitions. Consider having it before both, and then don't forget it's second one as well that takes subject attributes.